WHITEPAPER.md

```diff
@@ -26,6 +26,8 @@ Throughout this proposal, we will be using some terms that are fairly specific t
 * **instrumentation**: association between a particular result with an change in the system’s condition (the method of measurement)
 * **calibration**: identification and refinement of the amount of uncertainty included in a result.
 * **observability**: capability of a system to be considered from different angles while it is being used
+* **blast radius**: the impact on the system from the experiment.
```

Comment:
I wouldn't say the "impact on the system" since you are trying to explore how big that impact can be through the experiment. The blast radius is more about a functional perimiter for organizational purpose. In other words, defining a blast radius means you communicate the level of service you intend on targetting.

Reply:
Wikipedia definition:
> the distance from the source that will be affected when an explosion occurs.

In a chaos engineering experiment context:
> the distance from the source that will be affected when an experiment occurs.

What do you think?